lib/authentication-context.js

```diff
@@ -202,6 +204,19 @@ AuthenticationContext.prototype._acquireToken = function(callback, tokenFunction
   });
 };
 
+AuthenticationContext.prototype._acquireCode = function (callback, codeFunction) { 
```

Comment:
I think you might want to call this method _acquireUserCode to disambiguate user code from authorization code.